frontend/dockerfile/docs/reference.md

```diff
@@ -829,6 +833,25 @@ RUN --mount=type=secret,id=aws,target=/root/.aws/credentials \
 $ docker buildx build --secret id=aws,src=$HOME/.aws/credentials .
 ```
 
+#### Example: Mount as environment variable
+
+The following example takes the secret `API_KEY` and mounts it as an
+environment variable with the same name.
+
+```dockerfile
+# syntax=docker/dockerfile:1
+FROM alpine
+RUN --mount=type=secret,id=API_KEY,env=API_KEY \
+    printenv API_KEY
```

Comment:
Maybe not print the API_KEY, as it would be saved in the history record logs then 😉

Reply:
I couldn't think of an example usage 😆 I can change it to some pseudo command

```
RUN --mount=type=secret,id=api-token,env=API_KEY \
    some-command --token-env API_KEY
```